packages/salesforcedx-apex-debugger/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-apex-debugger",
+  "displayName": "Apex Debugger Adapter",
+  "description": "Implements VSCode debug protocol for Apex Debugger",
```

Comment:
Oh, ok. In that case, how about, "Implements the VS Code Debug Protocol for the Apex Debugger"?